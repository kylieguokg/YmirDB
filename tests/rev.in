rev notExist
set a
get a
rev a
get a
append a 1
get a
rev a
get a
set a 1 2 3
get a
rev a
get a
push a 4
get a
rev a
get a
set b a
type b
rev b
bye
