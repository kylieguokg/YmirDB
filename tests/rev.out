> no such key

> ok

> []

> ok

> []

> ok

> [1]

> ok

> [1]

> ok

> [1 2 3]

> ok

> [3 2 1]

> ok

> [4 3 2 1]

> ok

> [1 2 3 4]

> ok

> general

> Simple entry only

> bye
